Modules/Options/GeneralTab/QuestieOptionsGeneral.lua

```diff
@@ -726,7 +762,7 @@ function QuestieOptions.tabs.general:Initialize()
                 type = "toggle",
                 order = 6,
                 name = function() return l10n('Auto Complete'); end,
-                desc = function() return l10n('Enable or disable Questie auto-complete quests.'); end,
+                desc = function() return l10n('Enable or disable Questie auto-completing quests.'); end,
```

Comment:
Remember to change the translation-key as well:

https://github.com/Questie/Questie/blob/35d09da8f998e74ea47f88dfd86ec2a059a918a4/Localization/Translations/Options/General.lua#L929

Reply:
done, will update the PR with the added strings